common/src/Assets/EntityModel.cpp

```diff
@@ -463,15 +463,21 @@ std::unique_ptr<Renderer::TexturedIndexRangeRenderer> EntityModelSurface::buildR
   const size_t skinIndex, const size_t frameIndex)
 {
   assert(frameIndex < frameCount());
-  assert(skinIndex < skinCount());
+
+  // If an out of range skin is requested, use the first skin as a fallback
+  auto actualSkinIndex = skinIndex;
```

Comment:
Please use a ternary and make the variable const, i.e. `const auto actualSkinIndex = skinIndex < skinCount() ? skinCount : 0;`

Reply:
No worries, wasn't sure of the code standards for a ternary so I avoided it to be safe.